test/fixtures/message/assert_throws_stack.snapshot

```diff
@@ -19,16 +19,16 @@ AssertionError [ERR_ASSERTION]: Expected values to be strictly deep-equal:
   generatedMessage: true,
   code: 'ERR_ASSERTION',
   actual: Error: foo
-      at assert.throws.bar (*assert_throws_stack.js:*)
-      at getActual (node:assert:*)
-      at Function.throws (node:assert:*)
-      at Object.<anonymous> (*assert_throws_stack.js:*:*)
       at *
       at *
       at *
       at *
       at *
-      at *,
+      at *
+      at *
+      at *
+      at *
+      at node:internal*main*run_main_module:*:*,
```

Comment:
This part of the stack trace might very well change in the future as we continue to merge the CommonJS and ESM module loaders. Can we find a way to filter out any stack trace references to the module loader(s)?

Reply:
I guess we can `str.replace(/node:internal.*/, '*')` or something?
I want the diff of this PR to be minimal so perhaps a followup PR can handle this kind of changes